xml/System.Net/Cookie.xml

```diff
@@ -113,11 +113,11 @@
   
  The `value` parameter for a <xref:System.Net.Cookie> must not be a `null` reference (Nothing in Visual Basic). The following characters are reserved and cannot be passed in the `value` parameter unless the string passed in the `value` parameter is enclosed in double quotes: semicolon, comma. So the following example constructor would succeed but when you try to add this <xref:System.Net.Cookie> to a <xref:System.Net.CookieContainer> instance with the <xref:System.Net.CookieContainer.Add%2A> or <xref:System.Net.CookieContainer.Add%2A> methods the operation will fail and throw an exception:  
```

Comment:
1. Comma after "succeed" and before "the operation will fail"
2. To disambiguate constructors, the first is [CookieContainer.Add(Cookie)]<xref:System.Net.CookieContainer.Add(System.Net.Cookie)>, and the second is [CookieContainer.Add(Uri,Cookie)]<xref:System.Net.CookieContainer.Add(System.Uri,System.Net.Cookie).
Also, this is a nit from the original remarks used throughout these topics, but a good change to make would be, instead of "the following characters....: semicolon and comma", to write "the semicolon (";") and comma (",") characters..."